README.md

```diff
@@ -1,4 +1,224 @@
+- The Operator currently runs on GKE.
+- Currently the Kubernetes CA is required to deploy a secure cluster.
+- Expansion of persistent volumes is not yet functional.
+
+## Prerequisites
+
+- Kubernetes 1.8 or higher
+- [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+- A GKE cluster
+
+## Install the Operator
+
+Locate the latest release tag [here](https://github.com/cockroachdb/cockroach-operator/tags).
+
+Clone the release tag:
+
+```
+git clone --depth 1 --branch <tag_name> https://github.com/cockroachdb/cockroach-operator
+```
+
+Apply the CustomResourceDefinition (CRD) for the Operator:
+
+```
+kubectl apply -f config/crd/bases/crdb.cockroachlabs.com_crdbclusters.yaml
+```
+
+Apply the Operator manifest:
+
+```
+kubectl apply -f manifests/operator.yaml
+```
+
+Validate that the Operator is running:
+
+```
+kubectl get pods
+```
+
+```
+NAME                                  READY   STATUS    RESTARTS   AGE
+cockroach-operator-6f7b86ffc4-9ppkv   1/1     Running   0          54s
+```
+
+## Start CockroachDB
+
+Download [`example.yaml`](https://github.com/cockroachdb/cockroach-operator/examples/example.yaml) and optionally modify the default configuration as outlined below.
+
+```
+apiVersion: crdb.cockroachlabs.com/v1alpha1
+kind: CrdbCluster
+metadata:
+  name: cockroachdb
+spec:
+  dataStore:
+    pvc:
+      spec:
+        accessModes:
+        - ReadWriteOnce
+        resources:
+          requests:
+          #   cpu: "4"
+          #   memory: "16Gi"
+            storage: 60Gi
+          # limits:
+          #   cpu: "4"
+          #   memory: "16Gi"
+        volumeMode: Filesystem
+  tlsEnabled: true
+  # nodeTLSSecret: cockroachdb.node
+  # clientTLSSecret: cockroachdb.client.root
+  image:
+    name: cockroachdb/cockroach:v20.1.4
+  nodes: 3
+```
+
+### Certificate signing
+
+By default, the Operator uses the built-in Kubernetes CA to generate and approve 1 client and 1 node certificate for the cluster.
+
+<!-- If you wish to use a non-Kubernetes CA, additionally enable the following lines in `example.yaml` and fill in the names of the corresponding node and client secrets:
+
+```
+nodeTLSSecret: <node-secret>
+clientTLSSecret: <client-secret>
+```
+
+For example, if you are using [`cockroach cert`](https://www.cockroachlabs.com/docs/stable/cockroach-cert.html) to generate certificates, use these secret names:
+
+```
+nodeTLSSecret: cockroachdb.node
+clientTLSSecret: cockroachdb.client.root
+``` -->
+
+### Set resource requests and limits
+
+Before deploying, we recommend explicitly allocating CPU and memory to CockroachDB in the Kubernetes pods. 
+
+Enable the commented-out lines in the `resources.requests` object and substitute values appropriate for your workload. Note that `requests` and `limits` should have identical values.
+
+```
+resources:
+  requests:
+    cpu: "4"
+    memory: "16Gi"
+    storage: 60Gi
+  limits:
+    cpu: "4"
+    memory: "16Gi"
+```
+
+### Apply the StatefulSet configuration
```

Comment:
This is applying a CR and not a statefulset.  The example.yaml is a CR, a custom resource.